src/Sarif.Multitool/TransformCommand.cs

```diff
@@ -117,6 +113,21 @@ public int Run(TransformOptions transformOptions)
             return 0;
         }
 
+        private bool ValidateOptions(TransformOptions transformOptions)
+        {
+            bool valid = true;
+
+            if (transformOptions.SarifOutputVersion != SarifVersion.OneZeroZero && transformOptions.SarifOutputVersion != SarifVersion.Current)
+            {
+                Console.WriteLine(MultitoolResources.ErrorInvalidTransformTargetVersion);
+                valid = false;
+            }
+
+            valid &= DriverUtilities.ReportWhetherOutputFileCanBeCreated(transformOptions.OutputFilePath, transformOptions.Force, _fileSystem);
```

Comment:
>ReportWhetherOutputFileCanBeCreated [](start = 37, length = 35)

Shouldn't this be checking the value of GetTransformedOutputFileName (on line 34) rather than OutputFilePath?

Reply:
Yes, fixed.

---
In reply to: [312284809](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312284809) [](ancestors = 312284809)